src/sphinx/formats/universal.rst

```diff
@@ -158,6 +158,12 @@ If you want to force local for the `tgz` output add this line:
   universalArchiveOptions in (Universal, packageZipTarball) := Seq("--force-local", "-pcvf")
 
 This will set the cli options for the `packageZipTarball` task in the `Universal` plugin to use the options ``--force-local`` and ``pcvf``.
+Be aware that the above line will overwrite the default options.  You may want to prepend your options, doing something like:
+
+.. code-block:: scala
+
+  universalArchiveOptions in (Universal, packageZipTarball) :=
```

Comment:
Would the second one work?  I am under the impression that extra options to `tar` need to be _prepended_ rather than appended, because the default switches include `-c` which must be followed by the tar-file name.  But I believe the appended options will be placed _before_ the tar-file name.  Am I wrong about that?